cpp/include/Ice/Communicator.h

```diff
@@ -199,129 +187,121 @@ namespace Ice
         /// @see Connection::setAdapter
         void setDefaultObjectAdapter(ObjectAdapterPtr adapter);
 
-        /// Get the implicit context associated with this communicator.
+        /// Gets the implicit context associated with this communicator.
         /// @return The implicit context associated with this communicator; returns null when the property
```

Comment:
should we use null or nullptr in the descriptions?

Reply:
nullptr in C++